src/org/thoughtcrime/securesms/loki/LokiPushNotificationManager.kt

```diff
@@ -0,0 +1,80 @@
+package org.thoughtcrime.securesms.loki
+
+import android.content.Context
+import okhttp3.*
+import org.thoughtcrime.securesms.util.TextSecurePreferences
+import org.whispersystems.libsignal.logging.Log
+import org.whispersystems.signalservice.internal.util.JsonUtil
+import java.io.IOException
+
+object LokiPushNotificationManager {
+    //const val server = "https://live.apns.getsession.org/"
```

Comment:
Would be nice to automatically switch this based on the current build configuration so we don't run the risk of accidentally releasing with things pointing to the dev server